### DISCUSSION TITLE
Integrate spatie laravel-permission package

### DISCUSSION THREAD

**Discussion by bnqtoan** [filamentphp/filament#820] [Ideas]
It would be nice if we can manage role & permission via  spatie/laravel-permission package.

**Comment by molcsab** [comment]
I am using the package with Filament and it works perfectly.

**Reply by bnqtoan** [reply]
@molcsab Thank you for your comment.
I just concern about it because filament use its user model, not default laravel user model.
But looks like it's not the problem now. I'll try it.

**Comment by shankhadevpadam** [comment]
Is it possible to populate the selected role in the edit page. 

![Screenshot](https://github.com/filamentphp/filament/assets/3838722/122ce19f-131c-4b26-99bd-560b17f50835)

**Reply by leandrocfe** [reply]
```php
Select::make('roles')
    ->multiple()
    ->relationship('roles', 'name')
```
?
